src/sync.js

```diff
@@ -237,8 +237,8 @@ class Sync {
       if (err) { log.error(err) }
       if (err && err.code === 'ENOSPC') {
         callback(new Error('The disk space on your computer is full!'))
-      } else if (err && err.message === REVOKED) {
-        callback(err)
+      } else if (err && err.status === 400) {
+        callback(new Error(REVOKED))
```

Comment:
Sorry, I'm messing up revocation with token renewal... So I suppose a poorly handled 400 error could lead to the client being disconnected from the remote cozy... Or am I wrong?